docs/modules/ROOT/pages/reference/cloud-usage-reporting.adoc

```diff
@@ -0,0 +1,41 @@
+= Cloud Based AppCat Services
+
+This page will explain how the cloud based services will be handled for the usage reporting.
+The structure, content and interval of these metrics are not always under our control.
+We need a system which is flexible in adapting to those parameters.
+
+== Data Flow Overview
+
+See the https://docs.central.vshn.ch/metered-billing-data-flow.html[Data Model and Flow] for a generic overview of the data flow.
+
+Each cluster that provides AppCat cloud services will also have a cronjob configured.
+This cronjob will periodically run the AppCat cloud collector.
+Any necessary metadata will need to be available for this cronjob.
+Either by hard-coding it for a given cluster, or by dynamically querying relevant sources.
+The period at which it will run will depend on the cloud services as we have to match their reporting frequencies.
+
+=== Example configuration
+
+This configuration is abridged from the https://github.com/vshn/billing-collector-cloudservices/blob/master/component/class/defaults.yml[existing config] and only contains example configs for the new Odoo 16 billing.
+
+[code:yaml]
+----
+parameters:
+  billing_collector_cloudservices:
+    billingEndpoint:
+      URL: https://odoo16.vshn.ch <1>
+      API_TOKEN: topsecret <2>
```

Comment:
On APPUiO Managed clusters, that would expose the token to write to odoo16 to the customers and they can potentially directly interact with odoo and write arbitrary data in it (or in the worst case, delete data)

Reply:
Yes this is a security concern that must be raised with Aldebaran. I'm pretty sure though the data cannot be deleted or updated once you send it into the odoo16 queue.

Reply:
@TheBigLee asked tobru about this. He is looking into it.